core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -316,6 +316,8 @@ case object KAFKA_2_3_IV1 extends DefaultApiVersion {
   val id: Int = 23
 }
 
+
```

Comment:
Looks like we didn't need this?

Reply:
Ack. Was trying to create a `KAFKA_LATEST` case object but ended up with a different approach.